extensions/Worker.Extensions.SignalRService/src/Properties/AssemblyInfo.cs

```diff
@@ -3,4 +3,4 @@
 
 ﻿using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
 
-[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.SignalRService", "1.7.0")]
+[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.SignalRService", "1.8.0")]
```

Comment:
The main usage of `SignalROptions` is to allow users to configure JSON serialization options for the host process, but we've already had similar options in the isolated process model. Most of the rest options in `SignalROptions` can be achieved in the application settings. 

In the future we might need to consider how to avoid a second deserialization and serialization of the user payload in the host process which makes the serialization options of the worker process don't work.